README.md

```diff
@@ -42,17 +83,14 @@ See [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_gui
 
 ## More information
 
-<!-- List out where the user can find additional information, such as working group meeting times, slack/IRC channels, or documentation for the product this collection automates. At a minimum, link to: -->
-
+- [Ansible network resources](https://docs.ansible.com/ansible/latest/network/getting_started/network_resources.html)
 - [Ansible Collection overview](https://github.com/ansible-collections/overview)
 - [Ansible User guide](https://docs.ansible.com/ansible/latest/user_guide/index.html)
 - [Ansible Developer guide](https://docs.ansible.com/ansible/latest/dev_guide/index.html)
 - [Ansible Community code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html)
 
 ## Licensing
 
-<!-- Include the appropriate license information here and a pointer to the full licensing details. If the collection contains modules migrated from the ansible/ansible repo, you must use the same license that existed in the ansible/ansible repo. See the GNU license example below. -->
-
 GNU General Public License v3.0 or later.
 
-See [LICENCE](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
+See [LICENSE](https://www.gnu.org/licenses/gpl-3.0.txt) to see the full text.
```

Comment:
Maybe we should rename that file to `COPYING`. Most copyright headers in modules/plugins/... mention `COPYING`, not `LICENSE`.

Reply:
Not sure what you are asking here. We don't own the license file in the link and in general, we need a Licensing section in the readme files because some collections won't have an open license like this ...

Reply:
Our copy of the file is currently called `LICENSE`, as in the link label: https://github.com/ansible-collections/community.network/blob/master/LICENSE

In all copyright headers we mention a file called COPYING though: https://github.com/ansible-collections/community.network/blob/master/plugins/modules/network/a10/a10_server.py#L7

(In ansible/ansible, the license file is called COPYING: https://github.com/ansible/ansible/blob/devel/COPYING)

Reply:
I don't have any strong views either way, though being consistent with the new general OSS standards could be good.

Reply:
I've been through most other collections and set them to `COPYING`.